spec.html

```diff
@@ -25,13 +25,14 @@ <h1>Syntax</h1>
       `import` ModuleSpecifier `;`
       <ins>`import` ImportClause FromClause WithClause `;`</ins>
       <ins>`import` ModuleSpecifier WithClause `;`</ins>
+      <ins>`export` ExportFromClause FromClause WithClause `;`</ins>
 
     WithClause :
-      <ins>`with` WithList</ins>
+      <ins>`with` WithEntries</ins>
 
-    WithList :
+    WithEntries :
       <ins>IdentifierName `:` StringLiteral</ins>
```

Comment:
Should we allow number/bigint, boolean, null, and undefined?

Reply:
Broke out into https://github.com/tc39/proposal-module-attributes/issues/59 ; let's continue discussing there.